enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -725,3 +754,100 @@ def mapping_kernel_manager(self):
             return self.parent
         except AttributeError:
             return None
+
+    def _capture_user_update_overrides(self, **kwargs):
+        whitelisted_override_keys = [
+            "KERNEL_EXTRA_SPARK_OPTS",
+            "KERNEL_LAUNCH_TIMEOUT",
+        ]  # TODO need to read this list from env variable
+        user_requested_env_overrides = self.user_update_overrides.get("env", {})
+        allowed_env_overrides = {}
+        for override_key in whitelisted_override_keys:
+            if override_key in user_requested_env_overrides:
+                self.log.info("Key exist in extra overrides..")
+                if override_key == "KERNEL_LAUNCH_TIMEOUT":
+                    allowed_env_overrides[override_key] = str(
+                        user_requested_env_overrides.get(override_key)
+                    )
+                else:
+                    allowed_env_overrides[override_key] = self.user_overrides.get(
+                        override_key, ""
+                    ) + user_requested_env_overrides.get(override_key)
+        return allowed_env_overrides
+
+    def set_user_extra_overrides(self, update_payload):
+        # TODO need to read this list from env variable
+        whitelisted_override_keys = ["KERNEL_EXTRA_SPARK_OPTS", "KERNEL_LAUNCH_TIMEOUT"]
```

Comment:
We're trying to move away from certain socially-sensitive terms, so I'd rather we not introduce other instances.  Perhaps something like:
```suggestion
        allowed_override_keys = ["KERNEL_EXTRA_SPARK_OPTS", "KERNEL_LAUNCH_TIMEOUT"]
```